engelsystem/src/test/kotlin/info/metadude/android/eventfahrplan/engelsystem/extensions/CallExtensionsTest.kt

```diff
@@ -0,0 +1,176 @@
+package info.metadude.android.eventfahrplan.engelsystem.extensions
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.moshi.Moshi
+import info.metadude.android.eventfahrplan.engelsystem.models.ShiftsResult
+import info.metadude.kotlin.library.engelsystem.EngelsystemService
+import info.metadude.kotlin.library.engelsystem.models.Shift
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import java.io.EOFException
+import java.net.HttpURLConnection
+
+class CallExtensionsTest {
```

Comment:
A test with invalid JSON data (both non-JSON data and valid JSON data that doesn't match the "schema") would be nice.

Reply:
:heavy_check_mark: Good idea. Done in 0834d19ca714e71a4ebb347a4ccbe518625b3bfe.